### PR TITLE
Docs reorg - Billing pages

### PR DESCRIPTION
**Pull request by ksigler7**
- [x] Taken all valuable information from existing pages and added it to the new category
- [x] Deleted old pages
- [x] Deleted old images from images folder
- [x] Added remaining images to the img folder
- [x] Screenshots updated
- [x] all images have an alt tag
- [x] all images are named according to style guide
- [x] Updated the glossary
- [x] Added redirects from deleted page to new content
- [x] Head data should have SEO title, keywords, and description
- [x] Content is in Jobs-to-be-done format
- [x] Update all in-content links and links to deleted pages
- [x] Added additional resources (aim for a link to the glossary, the blog and other jobs-to-be-done)
- [x] Added a TOC
~- [ ] Added a talk to Dewey/Olivia section~
- [x] title optimized for google search snippits
- [x] content optimized for google search snippits
- [x] keyword optimization for each category
- [x] addressed/closed any related issues in the backlog
- [ ] pages are have the correct weight
- [x] content has been run through grammarly

### CODE REVIEW

**Comment by ksigler7** [comment]
eh, I am unsure how important the weights are here - should these two pages go together? Let me know what you think

File: source/User_Guide/Settings/billing.md [hunk 1]
```diff
@@ -1,63 +1,127 @@
...
+
+Refunds are only applicable to the subscription charge on your account, not for overage or contact storage charges from the prior month. We hold our customers responsible for managing their account credit limits and contact storage.
+
+If you meet these conditions, please respond directly to the invoice email you have received, or email billing@sendgrid.com to begin the discussion about a possible refund.
+
+{% anchor h2 %}
+Changing your plan
+{% endanchor %}
+
+To change your plan, click the “Change Plan” button next to your [billing details](https://app.sendgrid.com/settings/billing). Click the plan you would like and follow the prompts.
```
Comment (Whatthefoxsays) [inline source/User_Guide/Settings/billing.md]: To change your plan, click **Change Plan** next to your [billing details](https://app.sendgrid.com/settings/billing). Click the plan you would like and follow the prompts.

File: source/User_Guide/Settings/reading_your_invoice.md [hunk 2]
```diff
@@ -37,21 +34,19 @@ Need to send invoices to multiple addresses? Consider creating a [role address](
 
 These are the total charges for services used **in the current and previous month**: 
 
-![]({{root_url}}/images/invoice_NL_usage.png)
+![Charge summary section]({{root_url}}/img/invoice_NL_usage.png)
```
Comment (Whatthefoxsays) [inline source/User_Guide/Settings/reading_your_invoice.md]: For some reason this image didn't show up. Is this the one you accidentally deleted?

**Comment by thinkingserious** [comment]
Hello @ksigler7,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**